02_lecture/homework/task2.ts

```diff
@@ -0,0 +1,15 @@
+function arrayMutateRemove<Type>(
+  arr: Array<Type>,
+  func: Function
```

Comment:
Function is not the best option here as far as you lose dynamic typization. What I suggest is to go for a bit more flexible solution
```suggestion
  func: (element: TArrayElement) => boolean
```

Reply:
changed it